### ISSUE TITLE
[Feat]: use selenium nightly build for client tests

### ISSUE THREAD

**Issue by KazuCocoa** [appium/appium#20104]
I wondered if we could use nightly selneium bindings build for our client's test somehow. This is not per PR like our code change, but like every night to check our selenium deps for example. The purpose is to notice possible regression by selenium's updates or give feedback to selenium if the test fails.

I'm currently not sure if this idea will help both. Perhaps Ruby/Python are easier to test this out (?)

https://github.com/SeleniumHQ/selenium/releases/tag/nightly

cc @titusfortner

**Comment by mykola-mokhnach** [comment]
It's already implemented for java client: https://github.com/appium/java-client/blob/b1f8bc9e529906bea353e7bb3bd2c9eabc767e45/.github/workflows/gradle.yml#L68

**Comment by KazuCocoa** [comment]
oh, i didn't know of that

**Comment by mykola-mokhnach** [comment]
@KazuCocoa would you like to add more updates on this issue or it could be closed?

**Comment by KazuCocoa** [comment]
yeah, the original idea was for Java, so can be closed. we'll be able to create new one for other clients when we want that
